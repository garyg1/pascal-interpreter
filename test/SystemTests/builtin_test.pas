program builtin_test;

Var
    a : integer;

begin
    a := 4;

    Writeln(sqrt(a));
    Writeln(sin(a));
    Writeln(cos(a));
    Writeln(exp(a));
    Writeln(ln(a));
end.