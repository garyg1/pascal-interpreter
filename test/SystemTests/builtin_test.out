2.0
-0.7568025
-0.6536436
54.59815
1.3862944
